src/index.html

```diff
@@ -1,13 +1,48 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
         content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
-</html>
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body class="page_body">
+    <main class="page_main">
+      <div 
+      class="card_block"
+      data-qa="card"
+      >
+        <div class="card_block__first_element">
+          <img src="./images/imac.jpeg" alt="windows" 
+            class="card_block__img">
```

Comment:
apply code styling recommendations all over the page
```suggestion
          <img 
            src="./images/imac.jpeg" alt="windows" 
            class="card_block__img"
          >
```